roles/installer/tasks/60_deploy_ocp.yml

```diff
@@ -37,15 +37,15 @@
 
     - name: Wait for Bootstrap Complete
       shell: |
-        /usr/local/bin/openshift-baremetal-install --dir {{ dir }} --log-level debug wait-for bootstrap-complete
+        /usr/local/bin/{{ cmd }} --dir {{ dir }} --log-level debug wait-for bootstrap-complete
       register: wait_for_bootstrap_result
       until: wait_for_bootstrap_result is succeeded
       retries: "{{ increase_bootstrap_timeout|default(1)|int }}"
       delay: 1
 
     - name: Wait for Install Complete
```

Comment:
May user command as var and use ternary so ansible will not complain?

`.   vars:
         command: "{{.release_version is version('4.16', ">=") and ansible_distribution_major_version == '8' | ternary('openshift-install', 'openshift-baremetal-install') }}"`
       ansible.builtin.command: "{{ command }}"